tm2/pkg/crypto/bip39/bip39.go

```diff
@@ -201,7 +201,7 @@ func addChecksum(data []byte) []byte {
 
 func padByteSlice(slice []byte, length int) []byte {
 	newSlice := make([]byte, length-len(slice))
-	return append(newSlice, slice...) //nolint:makezero
+	return append(newSlice, slice...)
 }
```

Comment:
You can rewrite this method as:

```go
func leftPadSlice(slice []byte, length int) []byte {
	sliceSize := len(slice)

	if sliceSize >= length {
		// Nothing to pad
		return slice
	}

	padding := make([]byte, length)
	copy(padding[length-sliceSize:], slice)

	return padding
}
```

This way there is always safety for padding length, and the nolint disappears :)